### PR TITLE
Add executeActionsEmail

### PR DESCRIPTION
**Pull request by JPRW10**

### CODE REVIEW

**Comment by fschmtt** [comment]
Hi, thanks for your contribution 🥳 

Do you mind adding some unit and integration test for this new endpoint?

There are a lot of examples already - let me know if I can assist you 😄

**Comment by JPRW10** [comment]
Hello!

I didn't add any unit or integration test because the endpoint is only sending a mail to specified user, basically there's nothing to test

Do you want me to add tests anyway?

**Comment by fschmtt** [comment]
Yeah would be nice if you could add them:
* The unit test makes sure there's no unwanted & undetected change.
* The integration tests are run against all Keycloak versions since 13.0.0 and ensure that they work with all of them.

Thank you 😄

File: tests/Integration/Resource/UsersTest.php [hunk 1]
```diff
@@ -167,6 +167,18 @@ public function testCreateUserWithPasswordCredential(): void
         static::assertNull($user);
     }
 
+    public function testExecuteActionsEmail(): void
```
Comment (fschmtt) [inline tests/Integration/Resource/UsersTest.php:170]: I just tested it out locally - calling this endpoint actually requires a few setup steps (setting a user email address, setting up SMTP).

You can leave out the integration test IMO, thanks for your attempt, though 😄

File: tests/Unit/Resource/UsersTest.php [hunk 2]
```diff
@@ -399,4 +399,25 @@ public function testRemoveRealmRoles(): void
 
         $users->removeRealmRoles('test-realm', 'test-user', $roles);
     }
+
+    public function testExecuteActionsEmail(): void
```
Comment (fschmtt) [inline tests/Unit/Resource/UsersTest.php:403]:
```suggestion
public function testExecuteActionsEmail(): void
    {
        $command = new Command(
            '/admin/realms/{realm}/users/{userId}/execute-actions-email',
            Method::PUT,
            [
                'realm' => 'test-realm',
                'userId' => 'test-user-id',
            ],
        );

        $commandExecutor = $this->createMock(CommandExecutor::class);
        $commandExecutor->expects(static::once())
            ->method('executeCommand')
            ->with($command);

        $users = new Users(
            $commandExecutor,
            $this->createMock(QueryExecutor::class),
        );

        $users->executeActionsEmail('test-realm', 'test-user-id');
    }
```

**Review by fschmtt — CHANGES_REQUESTED** [review]

**Review by fschmtt — APPROVED** [review]

**Comment by fschmtt** [comment]
Thanks again, use 0.15.0 to use your changes 🥳

**Pull request merged**